package.json

```
@@ -24,6 +24,7 @@
     }
   },
   "dependencies": {
+    "@evan.network/api-blockchain-core": "^2.3.0",
```

Why not use current version?
```suggestion
    "@evan.network/api-blockchain-core": "^2.13.0",
```